src/app/core/services/user-profile.service.js

```diff
@@ -12,7 +12,7 @@ export default function (app) {
           this.lastName = user.lastName || '';
           this.phone = user.phone || '';
           this.email = user.email;
-          this.role = usersMocksService.userRole;
+          this.role = 'role';
```

Comment:
I believe You will fix this hard code in next commits and You will use it from constants :)

Reply:
I have already created constants roles.USER and roles.ADMIN, but they will come in another PR.